Source/JavaScriptCore/heap/HeapCell.h

```diff
@@ -60,7 +60,7 @@ class HeapCell {
     }
     bool isZapped() const { return !*bitwise_cast<const uint32_t*>(this); }
 
-    bool isLive();
+    bool isPendingDestruction();
```

Comment:
It's probably worth describing what this means in a comment.

Reply:
Added a comment. Please let me know if it's not clear, thanks!